assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -84,8 +111,39 @@ const WPDashboardWidgets = () => {
 					) }
 				</div>
 			) }
+
 			{ analyticsModuleActive && analyticsModuleConnected && (
-				<WPDashboardPopularPagesWidget />
+				<Fragment>
+					<WPDashboardUniqueVisitorsWidget />
+					<div className={ classnames(
+						'googlesitekit-wp-dashboard-stats__area-wrapper',
+						{
+							[ HIDDEN_CLASS ]: shouldCombineAnalyticsArea1,
+						} ) }>
+						<WPDashboardSessionDurationWidget />
+					</div>
+				</Fragment>
+			) }
```

Comment:
You're changing here the order of the widgets it looks like. These two widgets used to be displayed on top if Analytics is active, that should not be changed. If Analytics is active, the two widgets should be on top - if Analytics is not active, the Analytics CTA should be below the Search Console widgets, so that still the actual data remains on top.

Can you move the updated clause back above where it used to be?

Reply:
Sure, sorry about that.